openapi.yaml

```diff
@@ -1,190 +1,432 @@
-  pageNumber:
-    name: page[number]
-    in: query
-    type: integer
-    required: false
-    description: Page number of results
-    default: 1
-    minimum: 1
-  pageSize:
-    name: page[size]
-    in: query
-    type: integer
-    required: false
-    description: Number of results to return
-    default: 25
-    maximum: 500
-    minimum: 1
+  parkIdPath:
+    name: parkId
+    in: path
+    description: Unique Park ID
+    required: true
+    type: string
+  parkBody:
+    name: parkBody
+    in: body
+    description: A park body object
+    required: true
+    schema:
+      $ref: '#/definitions/PostParkModel'
+  ownerBody:
+    name: ownerBody
+    in: body
+    description: An owner body object
+    required: true
+    schema:
+      $ref: '#/definitions/PostOwnerModel'
 definitions:
-  Meta:
-    properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPages:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
   SelfLink:
     properties:
       self:
         type: string
+        example: request_url_here
         format: url
         description: Self-link of current resource
-  PaginationLinks:
+  OwnerResource:
+    required: [id, type, attributes]
     properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
+      id:
         type: string
-        format: url
-        description: The last page of data
-      prev:
+        example: '0001'
+        description: A unique ID of a park owner
+      type:
         type: string
-        format: url
-        description: The previous page of data
-      next:
+        description: type of object, should be owner
+        example: owner
+      attributes: &ownerAttributes
+        required: [ownerName]
+        properties:
+          ownerName:
+            type: string
+            description: Name of park owner / organization
+            example: City of Corvallis
+  PostOwnerModel:
+    required: [type, attributes]
+    properties:
+      type:
         type: string
-        format: url
-        description: The next page of data
-  PetResource:
+        description: type of object, should be owner
+        example: owner
+      attributes: *ownerAttributes
+  ParkResource:
     properties:
       id:
         type: string
-        description: A unique ID of pet
+        example: '0001'
+        description: A unique ID of park
       type:
         type: string
-        enum: [pet]
-      links:
-        $ref: '#/definitions/SelfLink'
-      attributes:
+        description: type of object, should be park
+        example: park
```

Comment:
Instead of `example`, since the only value should be `park`, you could use `enum: [park]`